app/controllers/sitemaps_controller.rb

```diff
@@ -41,7 +41,8 @@ def update
                       notice: t(".success")
         end
       else
-        format.html { render :edit }
+        flash.now[:alert] = "Invalid parameter"
```

Comment:
Can we dynamically show what the error message was here ? I'm not sure if anything gets added to `@planning_application.errors` for example

Reply:
Unless it should never reach this block and in that case a "please contact support" message might be helpful to indicate an action can be taken by the user